nixos/modules/config/resolvconf.nix

```diff
@@ -114,6 +114,14 @@ in
         '';
       };
 
+      subscriberFiles = lib.mkOption {
+        type = lib.types.listOf lib.types.path;
+        default = [];
+        description = ''
+          Files written by resolvconf updates
```

Comment:
I would add `internal = true`, since it's only used to inform the dhcpcd module about these files.

Reply:
got it